influence_benchmark/stats/preferences_per_iteration.py

```diff
@@ -42,85 +35,92 @@ def calculate_top_n_average(data: List[Dict], n: int) -> tuple:
     sorted_trajectories = sorted(trajectory_averages, key=lambda x: x[0], reverse=True)
 
     # Take the top N trajectories
-    top_n = sorted_trajectories[:n]
-
-    # Calculate the average of the top N
-    avg = sum(pref for pref, _, _, _ in top_n) / len(top_n) if top_n else None
+    if n > 0:
+        top_n = sorted_trajectories[:n]
 
+        # Calculate the average of the top N
+        avg = sum(pref for pref, _, _, _ in top_n) / len(top_n) if top_n else None
+    else:
+        top_n = []
+        avg = None
```

Comment:
Why would you ever want to do a topN with n=0?

Reply:
You might want to not plot the topN trajectories but only the average

Reply:
removed in newest version